signing_service/signing_service/signing_service.py

```diff
@@ -384,6 +410,28 @@ def _get_authentication_challenge_signature(self, authentication_challenge: byte
             authentication_challenge,
         )
 
+    @staticmethod
+    def _get_signing_service_daily_transaction_sum_so_far() -> int:
+        thresholds_directory = (Path.cwd()).joinpath('daily_tresholds')  # pylint: disable=no-member
+        thresholds_directory.mkdir(exist_ok=True)  # pylint: disable=no-member
+        daily_file = datetime.datetime.now().strftime('%Y-%m-%d')
+        thresholds_directory.joinpath(daily_file).touch(exist_ok=True)  # pylint: disable=no-member
+        thresholds_directory = thresholds_directory.joinpath(daily_file)
+        if thresholds_directory.read_text() == '':  # pylint: disable=no-member
+            thresholds_directory.write_text('0')  # pylint: disable=no-member
+            return 0
+        try:
+            transaction_sum = int(thresholds_directory.read_text())  # pylint: disable=no-member
+        except ValueError:
+            thresholds_directory.write_text('0')  # pylint: disable=no-member
```

Comment:
I think in this case you should also erase current file content (the one that caused `ValueError`).

Reply:
I've just tested `.write_text()`, and it does erase the whole file content. I don't know if any special erasing is needed?

Reply:
If it is working like that than its OK. I have took a quick look into documentation of this method but couldn't find this info, thanks.